reference/informative/informC06_interpretation_of_initial_values.rst

```diff
@@ -8,4 +8,148 @@
 
   .. container:: infospec
 
-    **TODO**
\ No newline at end of file
+    .. container:: heading3
+        
+        Cycles within initial values
+   
+    As with other places, it's possible to set initial conditions which are valid in syntax but lacking in sense.
+    The examples below are technically valid CellML, but they will not lead to meaningful models.
+
+    .. code-block:: xml
+
+      <component name="cyclical_initialisation" >
+        <variable name="A" initial_value="B" units="dimensionless"/>
+        <variable name="B" initial_value="A" units="dimensionless"/>
+      </component>
+
+      <component name="self_initialisation" >
+        <variable name="C" initial_value="C" units="dimensionless"/>
+      </component>
+
+
+    .. container:: heading3
+
+        Best practice for constants, variables, and initial conditions
+    
+
+    - A :code:`variable` whose value needs to be set for only the *beginning* of a simulation is initialised using the :code:`initial_value` attribute. 
+      
+        - This is most frequently used for state variables (those whose value is found by solving a differential equation).
+        - It's possible - but not recommended - to use a variable reference with the :code:`initial_value` attribute.
+          This option remains in CellML 2.0 only to provide ease of migration from CellML 1.1 models, and may be discontinued in future versions.
+          The recommended best practice is to make use of connections between variables to move all "hardcoding" of parameters (including initial values and constants) into a top-level component for ease of access.
```

Comment:
Maybe we can just leave this sentence out? I don't think it's necessarily best practice to do this, just that _if you want to have them in a separate component_, then this is the best way to do it

Reply:
OK, will do.

Reply:
Done in 6a718c4